test/snapshots/markdownlint-cli2-test-fs.js.md

```diff
@@ -870,6 +870,22 @@ Generated by [AVA](https://avajs.dev).
       `,
     }
 
+## markdownlint-cli2-config-option-with-extends (fs)
+
+> Snapshot 1
+
+    {
+      exitCode: 2,
+      formatterCodeQuality: '',
+      formatterJson: '',
+      formatterJunit: '',
+      formatterSarif: '',
+      stderr: `ENOENT: no such file or directory, open '/home/koder/GitHub/markdownlint-cli2/test/home/koder/GitHub/markdownlint-cli2/configs/.markdownlint-cli2.jsonc'␊
```

Comment:
Paths like this will fail on other machines. I use RegExp in cases where it is necessary.

Reply:
If CI passed with this present, it means this is probably unused.

Reply:
See also my comment on onlyRunViaExec.

Reply:
These extra ones got generated while working on the code. Didn't realize they remained in the commit. I've removed unwanted ones.